documentation/specs/TabPanels.md

```diff
@@ -0,0 +1,163 @@
+# `TabPanels` Component Specification
+
+## Overview
+
+A `TabPanels` is a set of tabs that let users switch between multiple views within a related context under the same URL.
```

Comment:
**Nit:** The beginning of this sentence is slightly awkward because the component ends in `s`. I would either kill the `A` or add `component` but truly a nit comment.